src/rfb.rs

```diff
@@ -650,11 +655,12 @@ pub struct FramebufferUpdateRequest {
     resolution: Resolution,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Copy, Clone)]
 #[allow(dead_code)]
 pub struct KeyEvent {
-    is_pressed: bool,
-    key: Keysym,
+    pub is_pressed: bool,
+    pub keysym: Keysym,
+    pub keysym_val: u32,
```

Comment:
Small question, just for my reference: are callees expected to use this value (instead of the translated `Keysym`) for anything other than tracing/diagnostic purposes? (This might be a good place for a doc comment, since this struct is part of the public interface in the `Server` trait.)

Reply:
Yeah, the value is there for debugging only. I can add a comment.

Reply:
Worth just making this a method on `Keysym` itself (e.g. `key.raw()`)?

Reply:
How would you feel if it was a method on the KeyEvent instead? When I ended up adding this field to KeyEvent, I was trying to avoid having to write another giant-conversion-table function for KeySym (or embedding the value in the enum).

Reply:
Yea, maybe even just use accessor methods instead of public fields so we can possibly change things without breaking consumers

Reply:
done in 012ab7e